core/src/main/java/org/commonjava/indy/core/content/DefaultContentManager.java

```diff
@@ -147,7 +153,7 @@ public Transfer retrieveFirst( final List<? extends ArtifactStore> stores, final
                 List<ArtifactStore> members;
                 try
                 {
-                    members = storeManager.getOrderedConcreteStoresInGroup( store.getName() );
+                    members = storeManager.getOrderedConcreteStoresInGroup( store.getName(), false );
                 }
```

Comment:
As I explained in the commit message I filtered the stores to enabled only at places I was absolutely sure we need it, because it caused the problems I tried to resolve. At other places I left the original behaviour by using false because I was not sure at all where are those methods used and if it is desirable to filter stores or not. But the first impression here to me is also that they should be filtered although I would still check the call hierarchy to be at least 70% sure :-)